rules/subrepo_rules.build_defs

```diff
@@ -196,7 +198,9 @@ def github_repo(name:str, repo:str, revision:str, build_file:str=None, hashes:st
 
     prefix = f'{repo}-{prefix}'
 
-    headers = {}
+    headers = {
```

Comment:
I put this here just because it forces curl to be used even for public repos. I wanted to consistently use curl in my debugging. It's not necessary though.